src/js/sticky-header.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * This file creates the sticky functionality for the header/menu so that it scrolls up and down.
+ * To make it change colors add .sticky to the _header.scss and enjoy
+ * @author Jess McClary <jess.mcclary@tricomb2b.com>
+ */
+(function ($) {
+  'use strict';
+
+  // Adding background color to header for scrolling
+  $(window).on("scroll", function(){
+        if($(window).scrollTop() > 100) {
+            $("#header").addClass("sticky");
+        } else {
+            $("#header").removeClass("sticky");
+        }
+    });
+})(jQuery);
```

Comment:
I'm going to be a little nitpicky here, but for JS switch to single quotes unless doubles are specifically required. Also provide a variable here for the scrollTop switch value, so it's slightly easier to find and change.

Also the spacing is weird. Should be 2 spaces for indentation..